general/documentation-day.md

```diff
@@ -0,0 +1,29 @@
+# Documentation Day
```

Comment:
Sounds like a whole day of reading documentation 😛 What about `Documentation Review`?

Reply:
Documentation Review sounds pretty good. Then we should think about to rename the Readme Day as well ;) 

On the other hand "Documentation Day" and "Readme Day" sounds more like an "institution" than a (regular) task. The name should reflect the idea of the concept than what should be done.

Reply:
True, maybe I was biased in terms of selling this xD